scintilla/scintilla_changes.patch

```diff
@@ -74,7 +74,7 @@ index ed47aa8..e58f1ab 100644
  	LINK_LEXER(lmAsm);
 -	LINK_LEXER(lmAsn1);
 -	LINK_LEXER(lmASY);
--	LINK_LEXER(lmAU3);
+ 	LINK_LEXER(lmAU3);
```

Comment:
You also need to change the diff section header to match (`-77,123 +77,51`) as you're changing the final line count.

Reply:
I will check and fix it.